source/extensions/omni.isaac.lab/test/sensors/test_camera.py

```diff
@@ -360,6 +360,31 @@ def test_camera_set_world_poses_from_view(self):
         torch.testing.assert_close(camera.data.pos_w, eyes)
         torch.testing.assert_close(camera.data.quat_w_ros, quat_ros_gt)
 
+    def test_camera_get_world_poses(self):
```

Comment:
can you please also add a unit test for the local pose APIs?

Reply:
thanks for the suggestion. I'll add it soon

Reply:
added.